src/models/profile/profile.ts

```diff
@@ -6,21 +6,24 @@ export class Profile {
   primarySkill?: string;
   client?: string;
   funFact?: string;
+  buttonColor?: string;
+  showSolidButton?: boolean;
 
   constructor(
     firstName: string,
     lastName: string,
     avatarUrl: string,
     primarySkill?: string,
     client?: string,
-    funFact?: string) {
-      this.firstName = firstName;
-      this.lastName = lastName;
-      this.avatarUrl = avatarUrl || '';
-      this.primarySkill = primarySkill;
-      this.client = client;
-      this.funFact = funFact;
-    }
+    funFact?: string
+  ) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.avatarUrl = avatarUrl || '';
+    this.primarySkill = primarySkill;
+    this.client = client;
+    this.funFact = funFact;
+  }
 }
```

Comment:
profile should be passing in a profile object and not each individual attribute, but thats not part of this commit